File: 프로그래머스/unrated/181900. 글자 지우기/글자 지우기.java
import java.util.*;

class Solution {
    public String solution(String my_string, int[] indices) {
        String answer = "";
        
        // 1. my_string을 쉽게 접근할 수 있도록 문자열을 배열로
        String[] tmp = my_string.split("");
        
        //2. indices를 돌면서 지워야 할 값들을 비워둔다.
        for (int i = 0; i < indices.length; i++) {
            tmp[indices[i]] = "";
        }

        //3. 문자열을 하나로 합친다(빈 공간은 붙어도 어짜피 포함X, 단순 "")
        for (String x : tmp) {
            answer += x;
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181900. 글자 지우기/글자 지우기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string my_string, vector<int> indices) {
    sort(indices.begin(), indices.end());
    reverse(indices.begin(), indices.end());
    for(int i : indices){
        my_string.erase(i, 1);
    }
    return my_string;
}